src/internal/connector/sharepoint/collection_test.go

```diff
@@ -167,24 +167,6 @@ func (suite *SharePointCollectionSuite) TestCollection_Items() {
 	}
 }
 
-func (suite *SharePointCollectionSuite) TestCollectPages() {
```

Comment:
do we have other code that exercises the `collectPages` path (directly or indirectly)? Like an integration test or something?

Reply:
Oh, yes. This was a duplicate. `TestCollectPages` is in `/sharepoint/data_collections_test.go`.